dlp/api/DlpSample/Program.cs

```diff
@@ -125,64 +129,252 @@ class DeidFpeOptions : FpeOptions { }
+                    (DeleteTemplatesOptions opts) => InspectTemplates.DeleteInspectTemplate(opts.ProjectId, opts.TemplateName),
+                    (DeidMaskOptions opts) => DeIdentify.DeidMask(
+                        opts.ProjectId,
+                        opts.Value,
+                        opts.InfoTypes,
+                        opts.Mask,
+                        opts.Num,
+                        opts.Reverse),
+                    (DeidFpeOptions opts) => DeIdentify.DeidFpe(
+                        opts.ProjectId,
+                        opts.Value,
+                        opts.KeyName,
+                        opts.WrappedKeyFile,
+                        opts.Alphabet),
+                    (ReidFpeOptions opts) => DeIdentify.ReidFpe(
+                        opts.ProjectId,
+                        opts.Value,
+                        opts.KeyName,
+                        opts.WrappedKeyFile,
+                        opts.Alphabet),
+                    (ListJobsOptions opts) => Jobs.ListJobs(
+                        opts.ProjectId,
+                        opts.Filter,
+                        opts.JobType),
+                    (DeleteJobOptions opts) => Jobs.DeleteJob(opts.JobName),
+                    (CreateJobTriggerOptions opts) => JobTriggers.CreateJobTrigger(
+                        opts.ProjectId,
+                        opts.BucketName,
+                        opts.MinLikelihood,
+                        opts.MaxFindings,
+                        opts.ScanPeriod,
+                        opts.InfoTypes,
+                        opts.TriggerId,
+                        opts.DisplayName,
+                        opts.Description
+                    ),
+                    (ListJobTriggersOptions opts) => JobTriggers.ListJobTriggers(
+                        opts.ProjectId
+                    ),
+                    (DeleteJobTriggerOptions opts) => JobTriggers.DeleteJobTrigger(
+                        opts.TriggerName
+                    ),
+                    (NumericalStatsOptions opts) => RiskAnalysis.NumericalStats(
+                        opts.CallingProjectId,
+                        opts.TableProjectId,
+                        opts.DatasetId,
+                        opts.TableId,
+                        opts.TopicId,
+                        opts.SubscriptionId,
+                        opts.ColumnName
+                    ),
+                    (CategoricalStatsOptions opts) => RiskAnalysis.CategoricalStats(
+                        opts.CallingProjectId,
+                        opts.TableProjectId,
+                        opts.DatasetId,
+                        opts.TableId,
+                        opts.TopicId,
+                        opts.SubscriptionId,
+                        opts.ColumnName
+                    ),
+                    errs => 1);
+            }
+            catch (ArgumentOutOfRangeException)
+            {
+                Parser.Default.ParseArguments<
```

Comment:
Why is this inside of catch?

Reply:
...because it can only accept up to 16 command sets. 🙁